### PR TITLE
starter-journal-article:0.2.0

### PR DESCRIPTION
**Pull request by HPDell**
I am submitting
- [ ] a new package
- [x] an update for a package

Description: Make the template configurable via the argument `template`. Users can set self-defined templates for title, author list, institutions, abstract, and main text in this argument to override the default template.

### CODE REVIEW

File: packages/preview/starter-journal-article/0.2.0/README.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
+# A configurable starter template for journal articles
+
+This package provides a template for writing journal articles
+to organise authors, institutions, and information of corresponding authors.
+
+![](./thumbnail.png)
```
Comment (laurmaedje) [inline packages/preview/starter-journal-article/0.2.0/README.md:6]: I recommend not adding the thumbnail here as Typst Universe will show a click-to-enlarge thumbnail anyway.

File: packages/preview/starter-journal-article/0.2.0/template/main.typ [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#import "@preview/starter-journal-article:0.2.0": article, author-meta
+
+#show: article.with(
+    title: "Article Title",
```
Comment (laurmaedje) [inline packages/preview/starter-journal-article/0.2.0/template/main.typ:4]: There's one unnecessary extra level of indent here.

File: packages/preview/starter-journal-article/0.2.0/README.md (hunk 1 above)
Comment (HPDell) [reply packages/preview/starter-journal-article/0.2.0/README.md]: Thanks. This has been removed in cee5160deee5f229c2755c49dc4fa39cf62ae991 .

File: packages/preview/starter-journal-article/0.2.0/template/main.typ (hunk 2 above)
Comment (HPDell) [reply packages/preview/starter-journal-article/0.2.0/template/main.typ]: Thanks. The extra indent has been removed in cee5160deee5f229c2755c49dc4fa39cf62ae991 .

**Pull request merged**